### DISCUSSION TITLE
[BUG] Videos show blank screen on iOS 16.4

### DISCUSSION THREAD

**Discussion by soufianj2** [vimeo/player.js#997] [Bug reports]
### Expected Behavior
Video should play when started.

### Actual Behavior
The video shows a blank screen, the controls are visible and work, it starts showing again only if the quality is changed.

### Steps to Reproduce
Play any video on iOS 16.4

**Comment by rkrishnan8594** [comment]
We've not been able to reproduce this issue. Can you provide a reproducible example?

**Comment by tetek** [comment]
I see the same
- generate embed script
- create a website with just the iframe
- open on iOS 16.4 -> missing user icon and video preview
<img width="714" alt="Screenshot 2023-07-24 at 16 35 35" src="https://github.com/vimeo/player.js/assets/286878/eda5372c-139e-4875-b58e-0739431e5c7f">
- open on iOS 17, everything looks as expected 
<img width="427" alt="Screenshot 2023-07-24 at 16 34 32" src="https://github.com/vimeo/player.js/assets/286878/18e7e949-dcb6-44f8-81b2-de27ccd58a87">

**Comment by jdreetz** [comment]
> I see the same
> 
> * generate embed script
> * create a website with just the iframe
> * open on iOS 16.4 -> missing user icon and video preview
> 
> <img alt="Screenshot 2023-07-24 at 16 35 35" width="714" src="https://user-images.githubusercontent.com/286878/255622597-eda5372c-139e-4875-b58e-0739431e5c7f.png"> - open on iOS 17, everything looks as expected <img alt="Screenshot 2023-07-24 at 16 34 32" width="427" src="https://user-images.githubusercontent.com/286878/255622332-18e7e949-dcb6-44f8-81b2-de27ccd58a87.png">

@tetek could you share the model number you're seeing this issue on as well as a video id or test page that reproduces the issue?

**Comment by tetek** [comment]
@jdreetz, this is the exact iframe
```<iframe class="link-embed" width="100%" height="100%" src="https://player.vimeo.com/video/844876479?app_id=122963&amp;pip=false" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; web-share; fullscreen" allowfullscreen=""></iframe>```

iPhone 14 Pro with iOS 16.2, but also on a webkit embedded in my app
<img width="430" alt="Screenshot 2023-07-25 at 8 59 27" src="https://github.com/vimeo/player.js/assets/286878/7f707df7-1468-45a9-ba52-b7e406a30411">

**Comment by brandonhrowe** [comment]
@tetek Can you confirm if this is occurring on a real device or just Simulator? I'm able to reproduce in Simulator, but playback works as expected when I test with the same specs (iOS 16.2, iPhone 14 Pro) in a virtual device. We want to rule out if it's just a quirk with that iOS in Simulator.
